src/main/java/bio/terra/stairway/NamedParameterPreparedStatement.java

```diff
@@ -99,4 +100,8 @@ public void setString(String name, String value) throws SQLException {
   public void setInstant(String name, Instant value) throws SQLException {
     preparedStatement.setTimestamp(getIndex(name), Timestamp.from(value));
   }
+
```

Comment:
Adding a `setObject()` felt a little wild-west, so I restricted it to UUID.  I'd be fine with making it more general if others prefer.